media/css/firefox/channel.less

```diff
@@ -290,7 +292,9 @@ html[lang^="en"] {
 // Channel components
 
 .channel {
-    padding-top: @baseLine * 2;
+    padding-top: 80px;  /* It should be higher than the sticky .platform-nav so
```

Comment:
If we're going to do this I think we should probably make the nav smaller, as things only are *just* visible when the page scrolls to an anchor. It just doesn't look quite right, and there is already too much white space at the top of each section. The sticky nav also takes up far too much room on small viewports, so I think we should probably leave it alone there and only apply it to desktop.